flocker/provision/_install.py

```diff
@@ -834,10 +834,13 @@ def task_enable_docker(distribution):
     # Use the Flocker node TLS certificate, since it's readily
     # available.
     docker_tls_options = (
-        '--tlsverify --tlscacert=/etc/flocker/cluster.crt'
+        '--tls=true --tlscacert=/etc/flocker/cluster.crt'
```

Comment:
I think we should keep `'--tlsverify`. It ensures that docker daemon only accepts TLS connections from clients with a certificates signed by the Docker CA.

```
  --tls                                    Use TLS; implied by --tlsverify
  --tlscacert=~/.docker/ca.pem             Trust certs signed only by this CA
  --tlscert=~/.docker/cert.pem             Path to TLS certificate file
  --tlskey=~/.docker/key.pem               Path to TLS key file
  --tlsverify                              Use TLS and verify the remote
```